packages/auto-label/cloudbuild.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: gcr.io/cloud-builders/npm
+    id: "build"
+    waitFor: ["-"]
+    entrypoint: bash
+    args:
```

Comment:
I don't quite understand why we need `build-function.sh`.  Naively, I think this boils down to running:
```
npm install
npm run compile
```

Reply:
I didn't want to mess with the build/deploy scripts too much. It's using the same shell scripts but reduced to a single function. It's also copying the `package.json` but that may be a remnant from when we were not yet publishing `gcf-utils` and it was being referenced as a local package.

Reply:
If this is a step that can be further simplified, 👍

Reply:
Actually, it looks like the package.json is needed for GCF

Reply:
Yeah this is a neat corner I haven't thought much about.  I guess the "right" way to do this would be to `npm pack`, and deploy the contents of the tarball.  Honestly... what we have is fine.

Reply:
we tried using `npm pack` at the outset, and I believe ran  into some issues. This is back  when we were trying to  be fancy and symlink  `gcf-utils` ... so might be fine now.